python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_chaos_cantilever_beam_integration.py

```diff
@@ -32,7 +32,7 @@
 dist_E = cb.E
 dist_F = cb.F
 dist_L = cb.L
-dist_I = cb.II
+dist_I = cb.I
```

Comment:
This is wrong, it really is `II` according to [the code](https://github.com/openturns/openturns/blob/71d3b4e9905649bca18279da3135a5b6d2a49829/python/src/usecases/cantilever_beam.py#L72). May be to avoid a confusion of the candidate `I` name with the `L` letter, already used as the length of the beam?

Reply:
I modified it to follow flake8 recommendations

Reply:
I agree: we can delete this PR.